cloud/scope/machine.go

```diff
@@ -223,7 +228,7 @@ func (m *MachineScope) InstanceImageSpec() *compute.AttachedDisk {
 		version = *m.Machine.Spec.Version
 	}
 	image := "capi-ubuntu-1804-k8s-" + strings.ReplaceAll(semver.MajorMinor(version), ".", "-")
-	sourceImage := path.Join("projects", m.ClusterGetter.Project(), "global", "images", "family", image)
+	sourceImage := path.Join("projects", m.ClusterGetter.NetworkProject(), "global", "images", "family", image)
```

Comment:
The machine's image has to come from the host project? That seems strange, but its probably a GCP behavior that I'm not understanding.

Reply:
Appear that you are correct, this should be unaltered.